test/libponyc/recover.cc

```diff
@@ -738,3 +738,19 @@ TEST_F(RecoverTest, CanWriteTrn_TrnAutoRecovery)
 
   TEST_COMPILE(src);
 }
+TEST_F(RecoverTest, CanAutorecover_Newref)
```

Comment:
there should be a blank line here.

Reply:
I think we said that we would add all tests that we can to runner tests in which case this should be a runner test as we are only checking that it compiles.

Reply:
OK, I've moved it.